docs/core/plugins/mail_crypt.md

```diff
@@ -325,7 +325,7 @@ passdb static {
   password = pass
   fields {
     crypt_global_public_key_file = ecpubkey.pem
-    crypt_global_private_key/main/private_key = <content of ecprivkey.pem>
+    crypt_global_private_key/main/private_key_file = inline:<content of ecprivkey.pem>
```

Comment:
does this need to be base64 encoded because of newlines?

Reply:
You tell me :) You're the one who wrote the original documentation saying this, I've just kept modifying the syntax.

Reply:
But currently we don't support multi-line values in config-rewrite, so I guess so. So `base64-encoded content of ecprivkey.pem`?

Reply:
I added a new commit with some other changes too.